openrtb_ext/request.go

```diff
@@ -192,9 +193,6 @@ func (pg *PriceGranularity) UnmarshalJSON(b []byte) error {
 			}
 			// Enforce that we don't read "min" from the request
 			pgraw.Ranges[i].Min = prevMax
-			if pgraw.Ranges[i].Min < prevMax {
-				return errors.New("Price granularity error: overlapping granularity ranges")
-			}
```

Comment:
If we are ignoring `"min"` json field

Reply:
Yeah. That condition will never be met with the `pgraw.Ranges[i].Min = prevMax` line right before it.